new/reclassify_amazon_linux_platform_family.md

```diff
@@ -0,0 +1,30 @@
+---
+RFC: unassigned
+Author: Sean OMeara <sean@chef.io>
+Status: Draft
+Type: Standards
+---
+
+# Title
+
+Reclassify Amazon Linux platform_family
+
+## Motivation
+
+   As time passes, Amazon Linux looks less and less like a RHEL clone.
+   Release version numbers do not match EL5, EL6, EL7.
+   Amazon is now a rolling release
+   Different package sets are available
+   Package names and contents are different
+   Service providers are different
+
+## Specification
+
+Change Amazon Linux platform_family from 'rhel' to 'amazon'
```

Comment:
I received a response from the Amazon Linux AMI team with permission to post it here for transparency:

We consider the Amazon Linux AMI to be a distant cousin in the extended family of Red Hat-inspired Linux offerings.  We don't have plans to converge on a specific set of packages from RHEL 6, RHEL 7, or any particular Fedora release, because our customers ask us for features and functionality that exists in all of those different distributions.  Additionally, the Amazon Linux AMI includes packages and features for which Amazon is the upstream.  This allows us to support the latest features of EC2 and AWS as they are launched.

Because the roadmap for the Amazon Linux AMI is driven very strongly by customer requests, we have drifted from RHEL and Fedora in some important ways, including the kernel version that we provide and when we change it, providing multiple versions of languages (PHP, Python, Ruby, etc.) in the OS repositories directly, and the "aggressiveness" that we show towards certain packages in terms of keeping close to the latest from upstream, even when that means a lot of churn.  Another major difference is that we are still using sysvinit/upstart while the latest RHEL and Fedora have moved to systemd.

Based on the above reasoning, we include ID_LIKE="rhel fedora" in the /etc/os-release file.